docs/user/templates.rst

```diff
@@ -316,20 +319,174 @@ renku-specific variables that are always available in templates, namely:
+    :width: 100%
+    :align: center
+    :alt: Error fetching custom templates
+
+If you think your template may be useful for the broader community, you can
+have more visibility by including it in the
+`community-contributed project templates repository <https://github.com/SwissDataScienceCenter/contributed-project-templates>`_.
+Feel free to push a pull request and we will validate it.
+
+If you are working in a dedicated RenkuLab deployment and your local
+community needs the templates, you should contact the administrators to
+include your repository in the RenkuLab template source through the
 `renku-values file <https://renku.readthedocs.io/en/latest/admin/index.html#create-a-renku-values-yaml-file>`_.
 
-We are currently working on adding full support for custom template
-repositories in RenkuLab so that you can manually specify external resources
-not included in the configuration file.
+
+Create shareable links with pre-filled fields
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Most of the time, you want your repository to be immediately available
+to the target audience, maybe even having some variables set to specific
+values. We know manually entering the same values, again and again, is a
+tedious process, so we come up with a solution to automatically pre-fill
+some of the fields in the project creation page.
+
+You can create a link containing metadata embedded in a query parameter.
+Once you enter that link in a browser, the UI will automatically fetch
+the required data and pre-fill the fields for you.
+
+To create a shareable link, start by filling in all the fields as you would
+do when creating a new project. Instead of clicking on `Create project`,
+click on the dropdown on the right side of the same button and then on
+`Create link`.
+
+You should see a modal where you can select which fields to include in the
+metadata. Some may not be clickable (no value provided), and others are
+deselected by default. The URL updates in real-time, and you can copy it
+for future reference or share it with others.
+
+.. image:: ../_static/images/templates_shareable_link.png
+  :width: 100%
+  :align: center
+  :alt: Custom template source
+
+.. note::
+
+  You can include any of the listed fields in the link, but you should be
+  careful when including namespace and visibility. Your user namespace cannot
+  be available to any other users, and group namespaces may require specific
+  permission. The visibility is generally tied to the namespace visibility,
+  although ``private`` should always be available since it is the most
+  restrictive one.
+  
+  It would be best to prefer fixed references for custom template
+  repositories, especially when selecting a template and providing values for
+  variables. This means commits and tags are a good choice, while branches
+  are not. Otherwise, the template or the variables may change in a later
+  version, resulting in a corrupted link.
+
+
+Programmatically compute shareable links
```

Comment:
This documentation is great and easy to understand, but I wonder if our user documentation is the best place to have it. Would it maybe be better to put it on Discourse and just say:

```
If you need to generate project-creation links yourself, you can see an example of how to do this in [Discourse](link to post).
```

Reply:
Maybe that part is a bit too technical, but I would still include it here since it was explicitly requested by one of the first users of this feature.